app/scripts/presenters/visit_presenter.js

```diff
@@ -22,7 +30,15 @@ define([
     },
 
     imageUrl: function () {
-      return this.hasValidExtractedImageUrl() ? this.extractedImageUrl : this.screenshot_url;
+      var url = this.hasValidExtractedImageUrl() ? this.extractedImageUrl : this.screenshot_url;
+
+      // it would be smarter to only crop screenshots if we're not in retina, but it's easier
+      // to let the imageProxy just handle it for now.
+      return imageProxy.crop(url, IMAGE_DISPLAY_WIDTH, IMAGE_DISPLAY_HEIGHT);
+    },
+
+    imagePosition: function () {
+      return (this.title.length % 2) ? 'left' : 'right';
```

Comment:
Yeah I think it's readable too, but the logic behind why we're using title this way may not be clear (it's so an item never changes sides). The reason for alternating them is a design decision to lend visual interest and rhythm. We're already changing image sizes based on "interestingness", so the button targets will be moving anyway.